dabl-meta.yaml

```diff
@@ -1,6 +1,6 @@
 catalog:
     name: da-marketplace
-    version: 0.0.2
+    version: 0.0.2.1
```

Comment:
this breaks the semantic versioning. why not make it 0.0.3?

https://semver.org/

Reply:
Leaving as `0.0.2` for now, as I plan to PR the triggers today, will change the version then.